index.bs

```diff
@@ -2520,7 +2525,6 @@ steps are:” followed by a list, or “The <code>|operation|(<var ignore>arg1</
 <pre class="grammar" id="prod-ReturnType">
     ReturnType :
         Type
```

Comment:
Is this needed anymore?

Reply:
Not really, but I left it in because maybe it's useful to distinguish that in the future? I can just fold it away if y'all think that would be better.

Reply:
Let's fold it away; I think a big win of this PR is unifying the two value spaces.